src/org/ticket/model/dao/iGenericDAO.java

```diff
@@ -0,0 +1,11 @@
+package org.ticket.model.dao;
+
+import java.util.List;
+
+public interface iGenericDAO<T> {
```

Comment:
Maybe capitalize the `i` in `iGenericDAO` (?). It is better as we only use **PascalCase** when writing classes/interfaces. Plus it is quit appealing to my eyes as it looks more C#-ish, not sure of yours.

`IGenericDAO` :+1: 
`iGenericDAO` :-1:

Reply:
Done!